README.md

```diff
@@ -12,7 +12,7 @@ You will also need to install the following on your development machine:
 - Ruby, RubyGems
 - Sass (`sudo gem install sass`)
 - Node.js (10.14.2 or higher is recommended) and NPM (see <https://nodejs.org/en/download/package-manager/>)
-- Go to project root, then run `npm install`
+- Go to project root, then run `npm ci`
```

Comment:
Did someone ask you for this change? If yes, I think we should create a separate card for this

Reply:
Yeah it was Mustofa. But you're right, it's better with a separate card

Reply:
@Khalidxx Could you please create it yourself? note that now we have SmartTrader project to assign a card. Thanks.